README.md

```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## A little about me
```

Comment:
Good job leaving a space after your headers. This helps to make the code more reader-friendly and will help to prevent rendering issues in future assignments.